plio/utils/covariance.py

```diff
@@ -37,12 +38,15 @@ def compute_covariance(lat, lon, rad, latsigma=10., lonsigma=10., radsigma=15.,
     """
     lat = math.radians(lat)
     lon = math.radians(lon)
-    
+
+    if semimajor_axis is None:
+        semimajor_axis = rad
+
     # SetSphericalSigmasDistance
     scaled_lat_sigma = latsigma / semimajor_axis
 
     # This is specific to each lon.
-    scaled_lon_sigma = lonsigma * math.cos(lat) / semimajor_axis
+    scaled_lon_sigma = lonsigma / (math.cos(lat) * semimajor_axis)
```

Comment:
I'm guessing the ISIS code was used as a basis for this hence the duplicate error

Reply:
Yeah, that's why the comments reference random function names.